mlflow/pyfunc/backend.py

```diff
@@ -318,8 +328,13 @@ def serve_stdin(
         model_uri,
         stdout=None,
         stderr=None,
+        model_config=None,
     ):
         local_path = _download_artifact_from_uri(model_uri)
+
+        command_env = os.environ.copy()
+        self._set_mlflow_config_env(command_env, model_config)
```

Comment:
How is this command_env used?

Reply:
like:

```

backend.execute(
                command,
                command_env,
                stdout=stdout,
                stderr=stderr, ...
            )
```

Reply:
source code: https://github.com/mlflow/mlflow/blob/8e9c24cb9c3e19e41777aea2e7589bc286483742/mlflow/utils/environment.py#L821

Reply:
But we didn't pass `command_env` here?

Reply:
oh sorry missed it. let me add it.